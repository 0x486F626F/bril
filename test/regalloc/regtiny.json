{
  "functions": [
    {
      "instrs": [
        {
          "dest": "r_3",
          "op": "const",
          "type": "int",
          "value": 5
        },
        {
          "args": [
            "r_3"
          ],
          "op": "print"
        }
      ],
      "name": "main"
    }
  ]
}
