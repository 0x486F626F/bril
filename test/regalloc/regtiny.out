main {
	__r01: int = const 5;
	print __r01;
}
